app/javascript/packages/verify-flow/context/verify-flow-context.tsx

```diff
@@ -0,0 +1,20 @@
+import { createContext } from 'react';
+
+interface VerifyFlowContextValue {
+  /**
+   * The path to which the current step is appended to create the current step URL.
+   */
+  basePath: string;
+
+  /**
+   * URL for reset password page in rails used for redirect
+   */
+  resetPasswordUrl: string;
+}
+
+const VerifyFlowContext = createContext({
```

Comment:
Can you merge this with `FlowContext`, since they're mostly for the same purpose.

https://github.com/18F/identity-idp/blob/main/app/javascript/packages/verify-flow/context/flow-context.tsx

No strong preference on which name we choose.